recipes-devtools/clang/compiler-rt-sanitizers_git.bb

```diff
@@ -15,12 +15,18 @@ LIC_FILES_CHKSUM = "file://compiler-rt/LICENSE.TXT;md5=d846d1d65baf322d4c485d6ee
 
 TUNE_CCARGS_remove = "-no-integrated-as"
 
-RUNTIME = "llvm"
+#compiler-rt-sanitizers will not compile for arm32 or arm64 based platforms without adding -fPIC
+CXXFLAGS:append = " -fPIC"
+
+#set RUNTIME variable according to documention to gnu as default value
+RUNTIME ?= "gnu"
```

Comment:
gnu is default runtime, so we don't need to repeat it here.

Reply:
Fine. Why has it been set to "llvm" previous to my changes though ?

Reply:
this recipe is derived from compiler-rt which obviously is part of llvm runtime but sanitizers could be decoupled from runtime, so I am ok removing it.